src/player.cpp

```diff
@@ -772,6 +772,15 @@ static void OnMapSaveFileReady(FileRequestResult*) {
 		Main_Data::game_data.party_location.position_x,
 		Main_Data::game_data.party_location.position_y
 		);
+
+	// Compatibility hack for old EasyRPG Player saves.
```

Comment:
Maybe we should add some version string, e.g. `(pre 0.6)` here for reference.

Reply:
Great idea. Fixed.